wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/shared/dto/IssueDto.java

```diff
@@ -35,4 +37,9 @@
     void setMessage(String message);
 
     IssueDto withMessage(String message);
+
+    static IssueDto create(Status status, String message) {
+        return newDto(IssueDto.class).withStatus(status)
```

Comment:
You should not use DtoFactory here. Because DTO is common class for client(gwt) and server. But gwt client doesn't have access to this class. I understand that gwt client doesn't use IssueDto. But any way it broke common concept. As far as I see you use it only twice in CodenvyLicenseManager. I think this method is redundant at all